app/lib/credentialDisplay/shared/utils/credentialSubject.ts

```diff
@@ -17,6 +17,7 @@ type CredentialRenderInfo = {
   numberOfCredits: string | null;
   achievementImage: string | null;
   achievementType: string | null;
+  alignments: Alignment[] | undefined;
```

Comment:
```suggestion
  alignment: Alignment[] | undefined;
```
The property name should probably be `alignment` singular (to match the OBv3 data model https://www.imsglobal.org/spec/ob/v3p0#achievement)? (In general, whenever possible, JSON-LD property names do not use plurals, because every property could always be singular OR an array.)